.github/workflows/tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Tests
+
+on:
+    push:
+    pull_request:
+
+jobs:
+
+    test:
+        name: PHPUnit Tests
+        runs-on: ubuntu-latest
+
+        env:
+            COMPOSER_ROOT_VERSION: dev-main
+
+        strategy:
+            matrix:
+                php:
+                    - '7.1'
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+            fail-fast: false
+
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    coverage: "none"
+                    extensions: "apcu, uuid"
```

Comment:
💡 We could play with this setting in the matrix, so run the tests with and without the polyfilled extensions.

Reply:
Not needed, that wouldn't provide anything